pkgs/development/python-modules/raylib-python-cffi/default.nix

```diff
@@ -9,9 +11,19 @@
   raylib,
   physac,
   raygui,
-  lib
+  darwin,
+  lib,
 }:
 
+let
+  inherit (darwin.apple_sdk.frameworks)
+    OpenGL
+    Cocoa
+    IOKit
+    CoreFoundation
+    CoreVideo
+    ;
+in
```

Comment:
The more common way to do this is to take each of these as arguments to the expression instead of `darwin` and then inherit them when calling this package similarly to here: https://github.com/mattpolzin/nixpkgs/blob/master//pkgs/top-level/python-packages.nix#L78..L80

Reply:
Hmm, perhaps not `raylib` seeing as how that would actually make it harder to move `raylib` into `pkgs/by-name`. Now I am second guessing whether the pattern I suggested to you is being phased out; let me see if I can find some PRs or issues related to this.

Reply:
Alright, I see plenty of recent activity that pulls `darwin` into packages in various ways so at the very least there's no need to standardize around what also might be "the old" way of bringing these dependencies in. Let's call my comment bad advice and leave things the way you've got them!